src/Analyzer/Utils.cpp

```diff
@@ -760,4 +760,52 @@ QueryTreeNodePtr createCastFunction(QueryTreeNodePtr node, DataTypePtr result_ty
     return function_node;
 }
 
+/** Returns:
+  * {_, false} - multiple sources
```

Comment:
The comment is not entirely correct. `_` is always `nullptr` and looks like it also happens if `getColumnSourceOrNull` returns `nullptr` (not multiple sources). Although it was just moved...